js/accordionView.js

```diff
@@ -0,0 +1,91 @@
+define([
+    'core/js/adapt',
+    'core/js/views/componentView',
+], function(Adapt, ComponentView) {
+
+    var AccordionView = ComponentView.extend({
+
+        events: {
+            'click .accordion-item-title': 'toggleItem'
+        },
+
+        preRender: function() {
+            // Checks to see if the accordion should be reset on revisit
+            this.checkIfResetOnRevisit();
+            this.listenTo(this.model, 'change:_items:_isActive', this.onItemsChanged);
+        },
+
+        postRender: function() {
+            this.setReadyStatus();
+        },
+
+        // Used to check if the accordion should reset on revisit
+        checkIfResetOnRevisit: function() {
+            var isResetOnRevisit = this.model.get('_isResetOnRevisit');
+
+            // If reset is enabled set defaults
+            if (isResetOnRevisit) {
+                this.model.reset(isResetOnRevisit);
+                this.model.resetItems();
```

Comment:
https://github.com/adaptlearning/adapt_framework/pull/1502/files#diff-5d38fecfdfca7e4c8accb31368a667caR8 < this line isn't needed as it'll have already be performed on line 28.